_includes/tags_list.html

```diff
@@ -0,0 +1,16 @@
+
+{% for tag in site.tags %}
+  {% assign t = tag | first %}
+  {% assign modules = tag | last %}
+
+<h3>{{ t | downcase }}</h3>
```

Comment:
What does this passage do? what does `| first` and `| last` mean?

Reply:
Hi Lorenzo. I believe these are Liquid filters - so they process the array of tags and store it in a variable.

Reply:
I just copied and pasted this code from some demo sites